Makefile

```diff
@@ -92,6 +92,9 @@ local-image-build: patch helm-lint helm-repo-index generate manifests-gen ## Bui
 local-image-push: ## Push docker image with the manager.
 	$(CONTAINER_COMMAND) push $(IMG)
 
+generate-mocks:
+	$(shell find . -name "mock_*.go" | grep -v vendor | xargs rm -f)
```

Comment:
Why delete files here? Won't they get overwritten anyway?

Reply:
just to make sure that all the mock are built again anew, i saw a post with recommendation that it is better to build them again.